domain/schema/controller.go

```diff
@@ -308,3 +312,158 @@ BEGIN
 END;
 `[1:]
 }
+
+func modelMigrationSchema() string {
+	return `
+CREATE TABLE model_migration (
+	uuid					TEXT PRIMARY KEY,
+	model_uuid				TEXT NOT NULL,
+	attempt					INT,
+	initiated_by_user_uuid	TEXT NOT NULL,
+	target_controller_uuid	TEXT NOT NULL,
+	target_cacert			TEXT,
+	target_entity			TEXT,
+	target_password			TEXT,
+	target_macaroons		TEXT,
+	active					BOOLEAN,
+	start_time				TIMESTAMP,
+	success_time			TIMESTAMP,
+	end_time				TIMESTAMP,
+	phase					TEXT,
+	phase_changed_time		TIMESTAMP,
+	status_message			TEXT
+	-- target_controller_alias
+	-- target_addresses
+	
+	-- TODO(anvial): Add foreign key constraints after corresponding tables are added.
+	-- 	CONSTRAINT				fk_migration_model
+	-- 		FOREIGN KEY			(model_uuid)
+	-- 		REFERENCES			model(uuid),
+	-- 	CONSTRAINT				fk_migration_initiated_by_user
+	-- 		FOREIGN KEY			(initiated_by_user_uuid)
+	-- 		REFERENCES			user(uuid),
+	-- 	CONSTRAINT				fk_migration_target_controller
+	-- 		FOREIGN KEY			(target_controller_uuid)
+	-- 		REFERENCES			controller(uuid)
+);
+
+CREATE TRIGGER trg_log_model_migration_insert
+AFTER INSERT ON model_migration FOR EACH ROW
+BEGIN
+    INSERT INTO change_log (edit_type_id, namespace_id, changed_uuid, created_at) 
```

Comment:
As it is, these are making change log entries for namespace ID 1, which is for external controllers. We need to figure out what entities are watched for model migrations and insert a new namespace(s) into `change_log_namespace` then use the ID(s) in these triggers.